src/pages/docs/getting-started/best-practices/partition-octopus-with-spaces.md

```diff
@@ -39,8 +39,8 @@ Internally we have opted for a space per application suite.
 We've also found several anti-patterns with the Spaces feature you should avoid.
 
 - A space per team (Team A Space, Team B space, etc.).  Typically in larger corporations, applications move between teams; a space per team would require you to move projects between spaces. The project export/import makes this easier, but it doesn't copy everything.  You'd need to move packages, deployment targets, and workers.  Release and Deployment history is not moved either.
-- A space per environment (Development Space, Production Space, Test Space, etc.).  Spaces were not designed, nor do they support this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.  Such a syncing process is [difficult to create and maintain](/docs/administration/sync-instances).
-- A space per tenant.  Just like the environments per space scenario, spaces were not designed, nor do they support this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.
+- A space per environment (Development Space, Production Space, Test Space, etc.).  Spaces were not designed for, nor do they support this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.  Such a syncing process is [difficult to create and maintain](/docs/administration/sync-instances).
+- A space per tenant.  Just like the environments per space scenario, spaces were not designed for, nor do they support this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.
```

Comment:
I was struggling w/ this the `for` is necessary, but a `,` is also to make the sentences work:
```suggestion
- A space per environment (Development Space, Production Space, Test Space, etc.).  Spaces were not designed for, nor do they support, this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.  Such a syncing process is [difficult to create and maintain](/docs/administration/sync-instances).
- A space per tenant.  Just like the environments per space scenario, spaces were not designed for, nor do they support, this scenario.  You would need a way to keep the deployment process in sync across multiple spaces.
```